OneCommunityGlobal/HighestGoodNetworkApp
# Olena - LessonList Component (route, figmaDesign)

# Description
Lesson list component for Phase II
<img width="680" alt="lessonlistpage" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49422649/28583fa9-f2fe-45f0-b12d-1051b1137a99">


## Related PRS (if any):
none
…

## Main changes explained:
-created route
-created the design for the Lesson List component based on the Figma specifications
-made the changes to the card nav bar due to Jae request


## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as any user
4. go to http://localhost:3000/bmdashboard/lessonlist
5. verify design & tag input functionality and expand/collapse card functionality


## Note:
At the moment, only the tag functionality is operational. 
Please note that the Figma navigation bar and the current design differ. I've made adjustments based on Jae's request, and he has also verified the design.



## Screenshots or videos of changes:




https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49422649/3bb09c8e-1aca-4164-818a-c9f915446282
